README.md

```diff
@@ -86,9 +86,16 @@ spec:
 
 ## Quick Start
 
-Clone the repository and run `make local` in an existing `kube:admin` OpenShift
-CLI session. 
+### Running operator locally
 
+Clone the repository and run `make local` in an existing `kube:admin` OpenShift CLI session.
+
+### Installing the operator in a cluster
+
+Start a kubernetes cluster. For this case we have used minikube. Any other Kubernetes cluster can be used as well.
+`minikube start`
+
+Now run the install script `sh install.sh`
```

Comment:
`install.sh` should have execution permission set, so that `install.sh` is just enough.

Reply:
done